_pages/JezBassinder.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: JezBassinder
+pageUrl: JezBassinder.github.io
+timestamp: TODAYS-DATE-2017-06-04
```

Comment:
@JezBassinder this suggestion by @badcoder28 is correct. Also add a line break at the end of your file and you should be all set!